setup.py

```diff
@@ -114,7 +114,7 @@
     "GitPython<3.1.19",
     "hf-doc-builder>=0.3.0",
     "hf_xet",
-    "huggingface-hub>=0.34.0,<1.0",
+    "huggingface-hub==1.0.0.rc0",
```

Comment:
I would suggest to merge this on `main` and once `v1.0` is released I'll open a follow-up PR to update to

```
"huggingface-hub>=1.0.0,<2.0",
```